docs/ivys.js

```diff
@@ -60,7 +60,9 @@ var emptyProductState = function (params) {
         // product tags eg 'tag-id' : 'tag-name'
         tags: {},
         // Collection name
-        collection: params.collection
+        collection: params.collection,
+        // Favorite status
+        favorite: false //true or false
```

Comment:
Probably not. You can look up the sku in the favorites object if you need to. This is just one more thing try to keep up to date.